testsuite/tests/kuadrant/authorino/wristband/conftest.py

```diff
@@ -0,0 +1,110 @@
+"""Conftest for Edge Authentication tests"""
+import pytest
+
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.certificates import CertInfo
+from testsuite.objects import Authorization
+from testsuite.utils import cert_builder
+
+
+@pytest.fixture(scope="module")
+def wristband_secret(blame, request, openshift, certificates) -> str:
+    """Create signing wristband secret"""
+    wristband_secret_name = blame("wristband-signing-key")
+    secret = openshift.create_signing_secret(wristband_secret_name, certificates["signing_ca"].key)
+    request.addfinalizer(lambda: openshift.delete_selector(secret))
+    return wristband_secret_name
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, wildcard_domain):
+    """Certificate hierarchy used for the wristband tests"""
+    chain = {
+        "signing_ca": CertInfo(ca=True),
+    }
+    return cert_builder(cfssl, chain, wildcard_domain)
+
+
+@pytest.fixture(scope="module")
+def envoy_template():
+    """Envoy with additional route match"""
+    return "envoy_wristband.yaml"
+
+
+@pytest.fixture(scope="module")
+def wristband_endpoint(openshift, authorino, authorization_name):
+    """Authorino oidc wristband endpoint"""
+    return f"http://{authorino.oidc_url}:8083/{openshift.project}/{authorization_name}/wristband"
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, wristband_secret, wristband_endpoint) -> Authorization:
+    """Add wristband response with the signing key to the AuthConfig"""
+    authorization.responses.add(
+        {
+            "name": "wristband",
+            "wrapper": "envoyDynamicMetadata",
+            "wristband": {
+                "issuer": wristband_endpoint,
+                "tokenDuration": 300,  # default value
+                "signingKeyRefs": [
+                    {
+                        "name": wristband_secret,
+                        "algorithm": "RS256",
+                    }
+                ],
+            },
+        }
+    )
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def wristband_token(client, auth):
+    """Test token acquirement from oidc endpoint"""
+    client.headers = {"Authorization": "Bearer " + auth.token.access_token}
```

Comment:
if you are using auth, you dont need to add headers manually, you can use `client.get("/auth", auth=auth)`